xgi/algorithms/centrality.py

```diff
@@ -307,3 +308,75 @@ def line_vector_centrality(H):
             vc[node].append(c_i[node])
 
     return vc
+
+
+def katz_centrality(H, index=False, cutoff=100):
+    """Returns the Katz-centrality vector of hypergraph H.
+
+    Parameters
+    ----------
+    H : xgi.Hypergraph
+        Hypergraph on which to compute the alpha centralities.
```

Comment:
Suggestion: we should use consistent naming all throughout the docs.

```suggestion
        Hypergraph on which to compute the Katz centralities.
```

Reply:
Fixed.